CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 ### unreleased
 
+### 0.3.0 (2020-09-01)
+
+* CI: remove (internal) DA Slack notifications (#750) - (Gary Verhaegen)
+* Add session-loader to hie.yaml (#714) - (Luke Lau)
+* Codeaction for exporting unused top-level bindings (#711) - (shaurya gupta)
+* Add links to haddock and hscolour pages in documentation (#699) - (Luke Lau)
+* Expose GHC.Compat module (#709) - (Pepe Iborra)
+* Move session loading logic into ghcide library (#697) - (Luke Lau)
+* Code action: remove redundant constraints for type signature (#692) - (Denis Frezzato)
+* Fix Binary instance of Q to handle empty file paths (#707) - (Moritz Kiefer)
+* Populate ms_hs_date in GetModSummary rule (#694) - (Pepe Iborra)
+* Allow GHC plugins to be called with an updated StringBuffer (#698) - (Alfredo Di Napoli)
+* Relax upper bounds for GHC 8.10.1 (#705) - (Pepe Iborra)
+* Obtain the GHC libdir at runtime (#696) - (Luke Lau)
+* Expect bench experiments to fail with Cabal (#704) - (Pepe Iborra)
+* Bump lodash from 4.17.15 to 4.17.19 in /extension (#702) - (dependabot[bot])
+* Update to hie-bios 0.6.1 (#693) - (fendor)
+* Backport HIE files to GHC 8.6 (#689) - (wz1000)
+* Performance improvements for GetSpanInfo (#681) - (Pepe Iborra)
+* Code action add default type annotation to remove `-Wtype-defaults` warning (#680) - (Serhii)
+* Use a global namecache to read `.hie` files (#677) - (wz1000)
+* Completions need not depend on typecheck of the current file (#670) - (Pepe Iborra)
+* Fix spaninfo Haddocks for local modules (#678) - (Pepe Iborra)
+* Avoid excessive retypechecking of TH codebases (#673) - (Pepe Iborra)
+* Use stale information if it's available to answer requests quickly (#624) - (Matthew Pickering)
+* Code action: add constraint (#653) - (Denis Frezzato)
+* Code action: add constraint (#653) - (Denis Frezzato)
```

Comment:
```suggestion
* Code action: add constraint (#653) - (Denis Frezzato)
```

Reply:
Oops, the changelog is auto generated from `git log` so I'm not sure how this duplicated commit ended up there!